Makefile

```diff
@@ -9,6 +9,18 @@ PYTHON ?= python
 PIP ?= $(PYTHON) -m pip
 RM  ?= rm
 
+
```

Comment:
Let's see if we can do _without_ this by removing any sandboxed tests. This was a hack that should not have been made. We should have been using pytest.